docs/datamodel/triggers.rst

```diff
@@ -0,0 +1,257 @@
+.. versionadded:: 3.0
+
+.. _ref_datamodel_triggers:
+
+========
+Triggers
+========
+
+Triggers allow you to define an expression to be executed whenever a given
+query type is run on an object type. The original query will *trigger* your
+pre-defined expression to run along with the original query. These can be
```

Comment:
maybe mention here that the triggers are executed in the same transaction, if any, as the triggering query? (obvious to us but worth making clear to users)

Reply:
Good suggestion!